File: apps/problems/max-elements-sum/boilerplate-full/function.cpp
#include <iostream>
#include <vector>
#include <string>
#include <algorithm>
#include <iterator>
#include <climits>

using namespace std;

##USER_CODE_HERE##

int main() {
	int size_nums;
		cin >> size_nums;
		vector<int> nums(size_nums);
		for(int i = 0; i < size_nums; i++) {
			cin >> nums[i];
		}

	int result = maxElementSum(nums);
	cout << result << endl;
	return 0;
}

File: apps/problems/max-elements-sum/boilerplate-full/function.rs
use std::io::{self, BufRead};
use std::str::FromStr;

##USER_CODE_HERE##

fn main() {
	let stdin = io::stdin();
	let mut input = stdin.lock().lines().map(|l| l.unwrap());
	let size_nums: usize = input.next().unwrap().parse().unwrap();
	let nums: Vec<i32> = input.take(size_nums).map(|s| s.parse().unwrap()).collect();
	let result = maxElementSum(nums);
	println!("{}", result);
}